package.json

```diff
@@ -26,10 +26,14 @@
   },
   "dependencies": {
     "@cypress/code-coverage": "^3.10.7",
+    "@emotion/react": "^11.11.1",
```

Comment:
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/137791947/04127fb1-d498-4314-a9a8-4e4411b28214)

We want the bottom install instead of the top for styled components

Reply:
Looking at other branches which has MUI installed, such as navbar, @mui/styled-engine-sc seems to not be used at all. And when installing it via npm there are dependency errors. uninstalling everything and keeping only @mui/material and styled-components seems to work fine so it will be kept like that.